ServerCore/Pages/_Layout.cshtml

```diff
@@ -26,15 +34,91 @@
                     <span class="icon-bar"></span>
                     <span class="icon-bar"></span>
                 </button>
-                <a asp-page="/Index" class="navbar-brand">ServerCore</a>
+                <a asp-page="/Index" class="navbar-brand">@(Event?.Name ?? "Microsoft Puzzles")</a>
             </div>
             <div class="navbar-collapse collapse">
                 <ul class="nav navbar-nav">
                     <li><a asp-page="/Index">Home</a></li>
-                    <li><a asp-page="/Puzzleday/Index">Puzzleday</a></li>
-                    <li><a asp-page="/Puzzlehunt/Index">Puzzlehunt</a></li>
-                    <li><a asp-page="/About">About</a></li>
-                    <li><a asp-page="/Contact">Contact</a></li>
+
+                    @if (Event != null)
+                    {
+                    <li class="dropdown">
```

Comment:
Should we indent from the curly brace @asyasky?

Reply:
Yes please. :) An automatic style sheet is on my todo list, but in the mean time ctrl+k+d fixes up most things (not sure about cshtml, but it does in cs for sure).